boards/Makefile.common

```diff
@@ -127,8 +134,14 @@ SIZE      ?= $(TOOLCHAIN)-size
 OBJCOPY   ?= $(TOOLCHAIN)-objcopy
 OBJDUMP   ?= $(TOOLCHAIN)-objdump
 
+# Set default flags to produce binary from .elf
```

Comment:
I don't like the mix of `?=` and `+=` in this section, I think that makes it very confusing to know which tools will behave which way.

As stated below, I think `OBJCOPY_FLAGS` should be removed from this list, and then given the behavior of the remaining tools, the rest can all be `+=` without limiting options for others.

Reply:
reason for mixing ?= and +=  is to minimize changes to existing boards. I'd make everything as ?= for consistency.